app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -336,5 +337,44 @@ def log_flashes
   rescue => e
     Rails.logger.error("Failed to log Flash Prototypes #{e.message}.", backtrace: e.backtrace)
   end
+
+  def log_document_type_metrics
+    return if in_progress_form.blank?
+
+    fd = in_progress_form.form_data
+    fd = JSON.parse(fd) if fd.is_a?(String)
+    additional_docs_by_type = get_doc_type_counts(fd.fetch('additionalDocuments', fd.fetch('additional_documents', [])))
```

Comment:
You may be able to pull the `.fetch` into a method to clean this up, but it's more of a nit 😄